### PR TITLE
Add platform section detailing enhanced compliance environments

### PR DESCRIPTION
**Pull request by ak-aiven**

### CODE REVIEW

**Comment by laysauchoa** [comment]
Thanks for your contribution. Here are some suggestions to improve the information in this article:

- Add links to the relevant information. There are some opportunity here to take the reader to the company website, for example, when mentioning about plans. I believe there is a lot of official information that you can link to your article, this helps also to help users to confirm what they are reading.
- Some of the terms are added here with their abbreviation and it is recommended at least when you are introducing in the article to explicitly say what the abbreviation means. This is important because compliances terms can be unfamiliar to many people.

What do you think about those points?

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
+Enhanced compliance environments (ECE)
+===========================================
+
+As a business that collects, manages, and operates on sensitive data that is protected by privacy and
+compliance rules and regulations – any vendor that assists with this collection, management and
+operation is subject to these same rules and regulations. Aiven is able to meet the needs of these
```
Comment (laysauchoa) [inline docs/platform/concepts/enhanced-compliance-env.rst:6]:
```suggestion
operation is subject to these same rules and regulations. Aiven meet the needs of these
```

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Cost of ECE
+----------------
+The cost of an enhanced compliance environment, beyond the initial eligibility requirements,
+is exactly the same as a standard Aiven deployment. All service costs are the same and all
+egress networking charges are still covered in an ECE. Customers can still take advantage of
+any applicable marketplaces and spend down their commit accordingly.
+
+Similarities to a standard environment
+----------------
+In many ways an ECE is the same as a standard Aiven deployment. All of Aiven’s tooling
```
Comment (laysauchoa) [inline docs/platform/concepts/enhanced-compliance-env.rst:36]:
```suggestion
In many ways, an ECE is the same as a standard Aiven deployment. All of Aiven’s tooling
```

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+- No internet access. Internet access cannot even be allow-listed as would be the case in standard private VPCs
+- VPCs cannot be automatically provisioned and must be manually configured by our networking team.
+- VPC peering or Privatelink connections must be manually approved on the Aiven end
+
+With these differences in mind, Aiven requires the following to provision an Enhanced Compliance
+Environment:
+
+- A CIDR block for all region/environment combinations. For example, if you have a development, QA and production environment and operate in 3 regions in each of those, we will need 9 CIDR blocks.
+The necessary peering information to enable the peer from our end. This differ between clouds:
```
Comment (laysauchoa) [inline docs/platform/concepts/enhanced-compliance-env.rst:50]:
```suggestion
The necessary peering information to enable the peer from our end. This differs between clouds:
```

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    * VPC ID
+GCP:
+    * GCP Project ID
+    * VPC Network Name
+Azure:
+    * Azure Tenant ID
+    * Azure App ID
+    * Azure VNet ID
+
+What compliance are covered?
```
Comment (laysauchoa) [inline docs/platform/concepts/enhanced-compliance-env.rst:63]:
```suggestion
Which compliances are covered?
```

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+----------------
+Although not exhaustive, Aiven is capable of supporting both HIPAA and PCI DSS compliance.
```
Comment (laysauchoa) [inline docs/platform/concepts/enhanced-compliance-env.rst:65]:
```suggestion
Although not exhaustive, Aiven is capable of supporting both HIPAA and PCI DSS compliances.
```

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+- Plan to operate in one or more of Amazon Web Services (AWS), Google Cloud Platform (GCP), or Microsoft Azure
+- Total monthly spend is greater than $5,000
+- An active Enterprise Support Contract
+
+Cost of ECE
+----------------
+The cost of an enhanced compliance environment, beyond the initial eligibility requirements,
+is exactly the same as a standard Aiven deployment. All service costs are the same and all
+egress networking charges are still covered in an ECE. Customers can still take advantage of
+any applicable marketplaces and spend down their commit accordingly.
```
Comment (laysauchoa) [inline docs/platform/concepts/enhanced-compliance-env.rst:32]:
```suggestion
any applicable marketplaces and spend down their commitment accordingly.
```

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+An enhanced compliance environment will run on Aiven managed infrastructure with the additional
+compliance requirement that no ECE VPC is shared and the managed account is logically separated
+from the standard Aiven deployment account. This decreases the blast radius of the environment
+to prevent inadvertent data sharing. Furthermore, users of an ECE **must** encrypt all data prior
+to reaching an Aiven service. As part of the increased compliance of the environment, enhanced logging
+is enabled for – ``stderr``, ``stout``, and ``stdin``.
+
+Who is eligible?
+----------------
+Enhance compliance environments, although similar to standard environments, involve added setup
```
Comment (dbeech) [inline docs/platform/concepts/enhanced-compliance-env.rst:19]: Enhance => Enhanced

File: docs/platform/concepts/enhanced-compliance-env.rst [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+Accountability Act (HIPAA) and the Payment Card Industry Data Security Standard (PCI DSS)
+compliances. If you require compliance beyond these please contact our sales department so we
+can better understand your specific needs. Additionally, we also offer an alternative deployment
+option -- :doc:`Bring Your Own Account (BYOA) </docs/platform/concepts/byoa>`.
+
+Migrating
+----------------
+Migrations to Aiven are fairly straightforward in general, but migrating to an ECE can add a
+tiny bit of complexity. If the migration is for a new service there are a few standard
+migration methods that will work – please contact sales at Sales@Aiven.io and a Solution Architect will be
```
Comment (dbeech) [inline docs/platform/concepts/enhanced-compliance-env.rst:76]: Personal preference maybe, but I don't like how the email address is capitalised. Could this be a mailto link instead?

**Review by dbeech — COMMENTED** [review]
All looks good. Only suggesting very minor changes - feel free to ignore

File: docs/platform/concepts/enhanced-compliance-env.rst (hunk 8 above)
Comment (ak-aiven) [reply docs/platform/concepts/enhanced-compliance-env.rst]: Thanks @dbeech. I don't like the capitalization either, but the linter breaks if I don't... Link is better anyway though.

**Review by laysauchoa — APPROVED** [review]
Thank you for your contribution 👍

**Pull request merged**